src/classes/BDI_Donations.cls

```diff
@@ -173,7 +176,8 @@ public class BDI_Donations {
                     dataImport.PaymentImportStatus__c = Label.bdiCreated;
                 } else {
                     // validate against our desired matching behavior (exact match, best match, new, etc.)
-                    if (!isValidMatchBehavior(dataImport, matchBehavior)) {
+                    // Elevate payments are specifically excluded
```

Comment:
Is the comment correct or is the logic incorrect? it looks like all non-elevate payments are excluded, not the other way as the comment suggests

Reply:
@KietTruongSFDO I revised the wording to be more clear.